src/controllers/auth.ctrl.ts

```diff
@@ -12,30 +12,34 @@ export class AuthCtrl {
     res: Response,
     next: NextFunction
   ): Promise<void> {
-    const { email, password, username } = req.body;
-    try {
-      const existingUser: IUserModel = await UserModel.findOne({
-        $or: [
-          {
-            username
-          },
-          {
-            email
-          }
-        ]
-      });
-      if (existingUser) {
-        const e: ConflictError = new ConflictError();
-        e.status = 409;
-        throw e;
-      }
-    } catch (e) {
-      return next(e);
-    }
+    const { email, password, role, username, firstName, lastName } = req.body;
+    // try {
+    //   const existingUser: IUserModel = await UserModel.findOne({
+    //     $or: [
+    //       {
+    //         username
+    //       },
+    //       {
+    //         email
+    //       }
+    //     ]
+    //   });
+    //   if (existingUser) {
+    //     const e: ConflictError = new ConflictError();
+    //     e.status = 409;
+    //     throw e;
+    //   }
+    // } catch (e) {
+    //   return next(e);
+    // }
```

Comment:
@jtongay thoughts on this? i'm losing the granularity of the status code being specific to conflict `409`, but its one less query to the DB